src/serializer/serializer.js

```diff
@@ -1842,34 +1841,43 @@ export class Serializer {
   init(filename: string, code: string, map?: string = "",
       sourceMaps?: boolean = false, onError?: (Realm, Value) => void) {
     // Phase 1: Let's interpret.
+    if (this.options.profile) console.time("Speculative Initialization");
     this.execute(filename, code, map, onError);
     if (this.logger.hasErrors()) return undefined;
     this.modules.resolveInitializedModules();
```

Comment:
The call to `resolveInitializedModules` is potentially expensive, and should get its own time tracking.